mucommander-core/src/main/java/com/mucommander/core/BrowseLocationThread.java

```diff
@@ -548,4 +550,15 @@ private void showDownloadDialog(AbstractFile file) {
         // Show confirmation/path modification dialog
         new DownloadDialog(mainFrame, fileSet).showDialog();
     }
+
+    private boolean isSftpWithPrivateKey() {
+        boolean sftpWithPrivateKey = false;
+        CredentialsMapping[] matchingCredentials = CredentialsManager.getMatchingCredentials(this.folderURL);
+        if (matchingCredentials.length > 0) {
+            boolean isSftp = matchingCredentials[0].getRealm().getScheme().equals("sftp");
```

Comment:
boolean -> var?

Reply:
It's a matter of style, personally I think `var` promotes write-only style of coding (it's convenient when you write the code but not when you read it). However I can change if that is preferred in the context of this project

Reply:
> It's a matter of style, personally I think `var` promotes write-only style of coding (it's convenient when you write the code but not when you read it). However I can change if that is preferred in the context of this project

I don't insist, however I think the code is more clearer and cleaner when `var` used in such cases.